x/profiles/keeper/keeper_app_links.go

```diff
@@ -37,6 +37,10 @@ func (k Keeper) SaveApplicationLink(ctx sdk.Context, link types.ApplicationLink)
 		),
 	)
 
+	k.AfterApplicationLinkSaved(ctx, link)
+
+	k.AfterApplicationLinkDeleted(ctx, link)
```

Comment:
```suggestion
```
`AfterApplicationLinkDeleted` hook should be removed here since the function is `SaveApplicationLink`.

Reply:
Fixed, thanks!